src/sass/layout/global.scss

```diff
@@ -1,3 +1,8 @@
+* {
+    max-width: 1500px;
```

Comment:
dlaczego dodales max-width do kazdego elementu w HTML???
* okresla wszystkie elementy - uwazam ze jest to zbyteczne :)

Reply:
ustawilem to dlatego ze grafika nie jest przystosowana do takich rozdzielczosci [https://nimb.ws/jky3il](url)

Reply:
[https://nimb.ws/jky3il](url)